tests/src/org.eclipse.jgit/org.eclipse.jgit/6.5.0.202303070854-r/gradle.properties

```diff
@@ -0,0 +1,2 @@
+library.version = 6.5.0.202303070854-r
+metadata.dir = org.eclipse.jgit/org.eclipse.jgit/6.5.0.202303070854-r/
```

Comment:
`metadata.dir` can be removed since it is not used in `build.gradle`?

Reply:
That file is generated by the `scaffold` task so if it is not needed, should be removed from that task.